README.md

```diff
@@ -4,7 +4,7 @@
 
 <p align="center">
   <a href="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-kaoto?style=for-the-badge" alt="Marketplace Version"/></a>
-  <a href="https://github.com/KaotoIO/kaoto/releases"><img alt="Kaoto UI version" src="https://img.shields.io/badge/Kaoto_UI-2.2.0-orange?style=for-the-badge"></a>
+  <a href="https://github.com/KaotoIO/kaoto/releases"><img alt="Kaoto UI version" src="https://img.shields.io/badge/Kaoto_UI-2.3.0-RC1-orange?style=for-the-badge"></a>
```

Comment:
not sure if I remember correctly but I think double dash will be needed here `2.3.0--RC1`

Reply:
effectively, updated

Reply:
Yes:

Single dash ![image](https://img.shields.io/badge/Kaoto_UI-2.3.0-RC1-orange?style=for-the-badge)

Double dash ![image](https://img.shields.io/badge/Kaoto_UI-2.3.0-RC1--orange?style=for-the-badge)